kickstart.sh

```diff
@@ -304,8 +321,6 @@ git="$(which_cmd git)"
 NETDATA_SOURCE_DIR=
 if [ ! -z "${git}" -a -x "${git}" ]
 then
-    SOURCE_DST="/usr/src"
-
     [ ! -d "${SOURCE_DST}" ] && run ${sudo} mkdir -p "${SOURCE_DST}"
 
     if [ ! -d "${SOURCE_DST}/netdata.git" ]
```

Comment:
The problem is that if you do `./kickstart.sh --src-dir /opt --install /opt` you will end up with a mess!
So, I thought it was better the source to be named `netdata.git` and the binary installation `netdata` and allow both in `/opt`.

Actually, I always try to name directories mirrored from git as `.git`. It helps me remember that this directory is something that can be updated with a `git pull`.